### ISSUE TITLE
[Accessibility]: Terminal output is not read in real time on Mac for VoiceOver

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#154027]
CC @isidorn

I have tried `VSCode` terminal on Mac with accessibility setting being turned on.
Currently, terminal live region output
 is not read by VoiceOver in realtime.
 I have tested this with the latest VSCode released version as well as the latest insider version.

**Comment by isidorn** [comment]
@jooyoungseo thanks for filling this issue, and sorry for the slow response I was on vacation.

For me the content of the terminal nicely gets read out on my mac.
Do you have exact reproducable steps for this? Can you still reproduce this with latest insiders?

Assigning to @Tyriar and @meganrogge for terminal.

**Comment by VSCodeTriageBot** [comment]
This issue has been closed automatically because it needs more information and has not had recent activity. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.

Happy Coding!

**Comment by Tyriar** [comment]
Related: https://github.com/microsoft/vscode/issues/144928

**Comment by meganrogge** [comment]
`document.activeElement.parentElement.classList.contains('xterm-accessibility-tree')` is false here when it should be true

![Screenshot 2022-12-12 at 7 31 39 PM](https://user-images.githubusercontent.com/29464607/207204158-fe6056ba-187a-4deb-8f47-7edf61aad13e.png)

**Comment by meganrogge** [comment]
On Windows, it reads `"list item 12 of 13"` etc. instead of the text content

**Comment by meganrogge** [comment]
@jooyoungseo does this always reproduce for you? It happens intermittently for me.

**Comment by Tyriar** [comment]
Upstream issue: https://github.com/xtermjs/xterm.js/issues/3247

**Comment by Tyriar** [comment]
Prototype for new view that could replace navigation mode all together: https://github.com/xtermjs/xterm.js/pull/4340

**Comment by meganrogge** [comment]
will be fixed with #169853

**Comment by jooyoungseo** [comment]
I believe the new prototype using `contenteditable` will be a game changer! Blind programmers used to rely on an external terminal due to inconvenient integrated terminal navigation. Looking forward to it!

**Comment by Tyriar** [comment]
@jooyoungseo great to hear that, let us know when it ends up shipping 🙂
